examples/amp-story-auto-ads.amp.html

```diff
@@ -143,7 +143,7 @@ <h1>fly-in-right</h1>
       </amp-story-grid-layer>
     </amp-story-page>
 
-    <amp-story-page id="fly-in-top-page">
+    <amp-story-page id="fly-in-top-page" block-ads>
```

Comment:
you don't like "next-page-no-ad" ? :-)

this gives us the flexibility to introduce "prev-page-no-ad", if we want to.
it's also much clear to pub where the ads are blocked.

Reply:
Done.